ArduCopter/config.h

```diff
@@ -168,8 +168,11 @@
  # define FS_EKF_THRESHOLD_DEFAULT      0.8f    // EKF failsafe's default compass and velocity variance threshold above which the EKF failsafe will be triggered
 #endif
 
-#ifndef EKF_ORIGIN_MAX_DIST_M
```

Comment:
Notionally this could break people's existing setups where they're overriding this in their config file (or in their hwdef).  Similarly for the alt.

Perhaps retain EKF_ORIGIN_MAX_DIST_M but set it from the new `KM` equivalent if it is not set.